Web.md

```diff
@@ -124,6 +124,103 @@ MIME type mismatch or `opaque` response types
 [reject](http://tc39.github.io/ecma262/#sec-rejectpromise) the Promise with a
 `WebAssembly.CompileError`.
 
+### Structured serialization of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is a
+[serializable object](https://html.spec.whatwg.org/multipage/infrastructure.html#serializable-objects), and will require changes to the
+[HTML Standard](https://html.spec.whatwg.org/multipage/infrastructure.html#serialization-steps) as follows:
+
+[StructuredSerializeInternal](https://html.spec.whatwg.org/multipage/infrastructure.html#structuredserializeinternal) adds a new step after step 12:
+
+* Otherwise, if *value* has a [[WebAssembly.Memory]] internal slot, then:
+  1. Let *size* be the number value.[[BufferObject]].[[ArrayBufferByteLength]].
+  2. If ! [IsSharedArrayBuffer](https://tc39.github.io/ecma262/#sec-issharedarraybuffer)(*value*.[[BufferObject]]) is true, then:
+    1. If *forStorage* is true, then throw a
+       "`[DataCloneError](https://heycam.github.io/webidl/#datacloneerror)`"
+       `[DOMException](https://heycam.github.io/webidl/#dfn-DOMException)`.
+    2. Otherwise:
+       1. Set *serialized* to { [[Type]]: "WebAssembly.Memory",
+          [[ArrayBufferData]]: value.[[BufferObject]].[[ArrayBufferData]],
+          [[ArrayBufferByteLength]]: size,
+          [[AgentCluster]]: the [current Realm Record](https://tc39.github.io/ecma262/#current-realm)'s corresponding [agent cluster](https://tc39.github.io/ecma262/#sec-agent-clusters) }.
+       2. Otherwise:
```

Comment:
I think this "Otherwise:" step is meant to apply to the not-SAB case in which case it should be unindented.  Also, the same comment about losing 'maximum' applies.  Moreover, looking at the rich preview I think the indenting is messed up in a few other places as well.